include/picongpu/versionFormat.cpp

```diff
@@ -24,8 +24,10 @@
 #include <boost/config.hpp>
 #include <boost/preprocessor/stringize.hpp>
 
+#if( PMACC_CUDA_ENABLED == 1 )
 #include <cuda.h>
 #include <mallocMC/mallocMC.hpp>
```

Comment:
because it's the responsibility of the CS PhD student to lead the effort to add more backends ;)